olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/model/PrefetchedPartitions.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <olp/dataservice/read/DataServiceReadApi.h>
+
+namespace olp {
+namespace dataservice {
+namespace read {
+namespace model {
+
+/**
+ * @brief Represents the result of a prefetch operation for partitions.
+ */
+class DATASERVICE_READ_API PrefetchedPartitions {
```

Comment:
`PrefetchPartitionsResult` - the name of the result struct is the name of the call, basically.